packages/@storefront/sort/test/unit/sort.ts

```diff
@@ -76,6 +76,19 @@ suite('Sort', ({ expect, spy, stub, itShouldBeConfigurable, itShouldProvideAlias
     });
   });
 
+  describe('onUpdate()', () => {
+    it('should extract and spread sorts into state', () => {
+      const state: any = sort.state = <any>{ a: 'b' };
+      const sorts = { c: 'd' };
+      const extractSorts = (sort.extractSorts = spy(() => sorts));
+
+      sort.onUpdate();
+
+      expect(extractSorts).to.be.called;
```

Comment:
This is assertion is probably redundant. You can't get `sort.state` to have the same `sorts` value if `extractSorts` isn't called.

Reply:
Noted, will remove.